app/api/dao/admin.py

```diff
@@ -65,7 +65,9 @@ def revoke_admin_user(user_id: int, data: Dict[str, str]):
         """
         admin_user_id = data["user_id"]
 
-        if user_id == admin_user_id:
+        users_list = UserModel.query.filter(UserModel.is_admin == True).count()
```

Comment:
```suggestion
        admins_list = UserModel.query.filter(UserModel.is_admin == True).count()
```

Reply:
would `admin_count` be better since it isn't a list anymore?

Reply:
yap yap